### PR TITLE
Load updateinfo when needed + filelists in upgrade command

### PR DESCRIPTION
**Pull request by kontura**
Follow up for: https://github.com/rpm-software-management/libdnf/pull/1556

I discovered this while working on enabling advisory CI tests.

### CODE REVIEW

File: dnf5/commands/upgrade/upgrade.cpp [hunk 1]
```diff
@@ -65,7 +65,16 @@ void UpgradeCommand::set_argument_parser() {
...
     auto & context = get_context();
+    context.update_repo_load_flags_from_specs(pkg_specs);
     context.set_load_system_repo(true);
+    bool updateinfo_needed = advisory_name->get_value().empty() || advisory_security->get_value() ||
+                             advisory_bugfix->get_value() || advisory_enhancement->get_value() ||
+                             advisory_newpackage->get_value() || advisory_severity->get_value().empty() ||
+                             advisory_bz->get_value().empty() || advisory_cve->get_value().empty();
+    if (updateinfo_needed) {
+        context.set_available_repos_load_flags(
+            context.get_available_repos_load_flags() | libdnf::repo::LoadFlags::UPDATEINFO);
```
Comment (j-mracek) [inline dnf5/commands/upgrade/upgrade.cpp:76]: Shell we load updateinfo as a default set?

**Review by m-blaha — APPROVED** [review]
Thanks a lot. With this merged the CI should be reliable again.

**Comment by m-blaha** [comment]
@j-mracek would you mind turning your comment into a jira issue? I'd like to merge this ASAP, because otherwise the CI is broken.

**Comment by j-mracek** [comment]
> @j-mracek would you mind turning your comment into a jira issue? I'd like to merge this ASAP, because otherwise the CI is broken.

Sure

**Pull request merged**